scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -196,7 +196,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 	                      <tr>
 	                        <td width="20%">{{comment.created_at.strftime('%Y-%m-%d')}}</td>
 	                        <td width="20%">{{comment.user_name}}:</td>
-	                        <td><strong>{{ comment.content }}</strong></td>
+	                        <td><strong>{{ comment.content | replace(",",", ") | replace(",  ",", ") }}</strong></td>
```

Comment:
Double replace is for taking into account the cases where there is already a space after the comma, so you don't get a double space